### PR TITLE
Remove duplicate import

### PR DESCRIPTION
**Pull request by pigletfly**
**What type of PR is this?**

Remove duplicate import in rs controller

/kind cleanup
/priority backlog

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
	None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
	None
```

### CODE REVIEW

File: pkg/controller/replicaset/replica_set.go [hunk 1]
```diff
@@ -37,8 +37,7 @@ import (
 	"time"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/api/core/v1"
```
Comment (hasheddan) [inline pkg/controller/replicaset/replica_set.go:40]: nitpick: is added import alias needed here?

Comment (pigletfly) [reply pkg/controller/replicaset/replica_set.go:40]: It's autoformated by vscode-go plugin.I think it's better to use it as a explicit alias

**Pull request closed without merging**